presto-benchmark-driver/src/main/java/com/facebook/presto/benchmark/driver/BenchmarkQueryRunner.java

```diff
@@ -266,7 +266,9 @@ private long getTotalCpuTime()
         for (URI server : nodes) {
             URI addressUri = uriBuilderFrom(server).replacePath("/v1/jmx/mbean/java.lang:type=OperatingSystem/ProcessCpuTime").build();
             String data = httpClient.execute(prepareGet().setUri(addressUri).build(), createStringResponseHandler()).getBody();
-            totalCpuTime += parseLong(data.trim());
+            if (!data.trim().isEmpty()) {
```

Comment:
It is a bit hacky. A cleaner solution might be to pass a boolean to BenchmarkQueryRunner to request collection of CPU times from the workers. In case of C++ workers, that boolean would be set to false and we would leave nodes empty. Another option is to add support for the URI above to Prestissimo. Should we straightforward. See presto-native-execution/presto_cpp/main/TaskResource.cpp